sentinel-ai-core/src/main/java/com/phonepe/sentinelai/core/model/Model.java

```diff
@@ -55,7 +57,8 @@ default CompletableFuture<ModelOutput> stream(
             List<AgentMessage> oldMessages,
             Map<String, ExecutableTool> tools,
             ToolRunner toolRunner,
-            Consumer<byte[]> streamHandler) {
+            Consumer<byte[]> streamHandler,
```

Comment:
Swap these two parameters so that the variable param (streamHandler) remains at the end. Generally makes code more readable